tutorial/README.md

```diff
@@ -0,0 +1,281 @@
+
+# TokenScript for your token
+These instructions will help you create a TokenScript interface for a smart contract.
+User interface code and business logic is organised in XML declarations for a TokenScript engine to render in a mobile wallet. Eg [AlphaWallet](https://alphawallet.com/).
+
+
+## Creating your token and UI widgets
+
+* Open studio.ethereum.org, and start the Coin project
+  * It shows the smart contract and front-end files, and runs a vm and the Coin files in the browser
+
+Although not conforming to the ERC20 interface, a few changes to the example code will have it demonstrable in AlphaWallet
```

Comment:
Why is it not conforming to erc20?

Reply:
It is the example from Ethereum Studio, intentionally simple. (They also have an erc721 token example, but would be too much for a first-timer learning TokenScript)